internal/cluster/validator.go

```diff
@@ -281,6 +281,9 @@ func (v *clusterValidator) printSufficientMastersCount(context *clusterPreproces
 }
 
 func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext) validationStatus {
+	if len(c.cluster.Hosts) < common.MinMasterHostsNeededForInstallation {
```

Comment:
Isn't it now a duplication of sufficient-master-count validation? Why do we need two validations that check the same thing?

Reply:
no, there are a few scenarios: 
1. 3 masters in a known state and one worker with insufficient - sufficient-master-count validation= true allHostsAreReadyToInstall= false
2. 2 masters & 2 workers are in a known state - sufficient-master-count validation= false, allHostsAreReadyToInstall= true